EmberMate/Components/ContextMenu.swift

```diff
@@ -34,8 +37,7 @@ class ContextMenu: NSObject, NSMenuDelegate {
     }
 
     @objc open func preferencesClicked(_ sender: NSMenuItem) {
-        NSApp.activate(ignoringOtherApps: true)
```

Comment:
I think you still need this to bring the settings window to the foreground

Reply:
Thanks!